docs/source/VUDRM-key-provision.md

```diff
@@ -505,3 +505,128 @@ The values are:
 - `cupertinostreaming-aes128-iv-include-in-chunklist`: Must be set to `false`.
 - `cupertinostreaming-aes128-key-format`: Must be set to `com.apple.streamingkeydelivery`.
 - `cupertinostreaming-aes128-key-format-version`: Must be set to `1`.
+
+### Vualto Wowza DRM API
+
+The Vualto Wowza DRM API is a small Docker web API application which runs on the Wowza server, allowing the provisioning of keyfiles through a simple REST interface.
+
+#### Running using Docker
+
+In this example, the API is exposed on port 9000 and the API key is set to some GUID:
+
+```
+docker run -d --restart always -p 9000:80 \
+-e WOWZADRMAPI_APIKEY=93640045-31f1-45c0-aa0d-b5682d7c9ac8 \
+-e WOWZADRMAPI_KEYFILESPATH=/mnt/keyfiles \
+-e WOWZADRMAPI_GRANDCENTRALURL=https://config.vudrm.tech/v1/clients \
+-e WOWZADRMAPI_KEYPROVIDERURL=https://keyprovider.vudrm.tech \
+-v /usr/local/WowzaStreamingEngine-4.7.7/keys:/mnt/keyfiles \
+--name wowza-drm-api vualto/wowza-drm-api:latest
+```
+
+#### Supported actions
+
+`<streamid>` must match the name of the Stream File on the Wowza server.
```

Comment:
Need a bit more info as LIVE does not have a "Stream File"

https://www.wowza.com/docs/how-to-secure-mpeg-dash-streaming-using-common-encryption-cenc#dash_cenc

> Key files are text files that are located in the [install-dir]/keys folder that match the stream name of the stream you're playing with the addition of a .key extension.
>For example, if you want to protect the live stream myStream, you would create a key file with the path [install-dir]/keys/myStream.key.
>To protect the on-demand stream sample.mp4, create a key file with the path [install-dir]/keys/sample.mp4.key.